components/octavia/values.yaml

```diff
@@ -0,0 +1,135 @@
+---
+
+
+labels:
+  # (nicholas.kuechler) TODO: remove this label override when
+  # https://review.opendev.org/c/openstack/openstack-helm/+/952722
```

Comment:
If you bump the chartVersion to one of the 2025.1 charts it should have your change and you can drop this.